packages/executors/graphql-ws/src/index.ts

```diff
@@ -13,6 +13,10 @@ const defaultPrintFn = memoize1(print);
 interface GraphQLWSExecutorOptions extends ClientOptions {
   onClient?: (client: Client) => void;
   print?: typeof print;
+  /**
+   * Additional headers to include when querying the original schema
```

Comment:
The headers are only included in the upgrade request and never again during the lifecycle of the socket. I think it's important to emphasize this.